src/pages/docs/intro/quickstart.md

```diff
@@ -75,10 +75,8 @@ const user = await db.users.get(userId);
 
 // Might be null
 if (user) {
-  user.data.name;
-  //=> "Sasha"
-  user.ref.id;
-  //=> "ykodM19iSxnI9CG0nq3g"
+  user.data.name; // "Sasha"
+  user.ref.id;    // "ykodM19iSxnI9CG0nq3g"
```

Comment:
I prefer the original format I used for it in date-fns, and I think it works well replicating REPL.

```suggestion
  user.data.name;
  //=> "Sasha"
  user.ref.id;
  //=> "ykodM19iSxnI9CG0nq3g"
```